eduiddashboard/vccs.py

```diff
@@ -108,18 +108,18 @@ def add_credentials(vccs_url, old_password, new_password, user):
 
     if old_factor:
         vccs.revoke_credentials(str(user.user_id), [old_factor])
-        user.passwords.remove(checked_password.id)
+        user.passwords.remove(checked_password.credential_id)
```

Comment:
when modifying the element list, checked_password.key should be used. Credential ID refers to the credentials ID in the authentication backend database and .key just happens to equal .credential_id in the current implementation.